src/lib/Preview.js

```diff
@@ -163,6 +166,13 @@ class Preview extends EventEmitter {
         this.navigateLeft = this.navigateLeft.bind(this);
         this.navigateRight = this.navigateRight.bind(this);
         this.keydownHandler = this.keydownHandler.bind(this);
+
+        this.on(PREVIEW_ERROR, (msg) => {
```

Comment:
Arrow function is unnecessary here

Reply:
This section is for debugging purposes only